src/rules/createFontFaceRule.js

```diff
@@ -0,0 +1,20 @@
+import createBaseRule from './createBaseRule'
+import {toCSS} from '../utils'
+
+/**
+ * Create a font-face rule.
+ *
+ * @return {Object}
+ */
+export default function createFontFaceRule(selector, style, options) {
+  const rule = createBaseRule('font-face', {selector, style, options})
+
+  /**
+   * Generates a CSS string.
+   *
+   * @see toCSS
+   */
+  rule.toString = (options) => toCSS(rule, options)
```

Comment:
I think assigning functions like that is a perf killer in general, but I am not sure.